cmd/lncli/cmd_macaroon.go

```diff
@@ -212,3 +214,59 @@ func listMacaroonIDs(ctx *cli.Context) error {
 	printRespJSON(resp)
 	return nil
 }
+
+var deleteMacaroonIDCommand = cli.Command{
+	Name:      "deletemacaroonid",
+	Category:  "Macaroons",
+	Usage:     "Delete a specific macaroon ID.",
+	ArgsUsage: "root_key_id",
+	Description: `
+	Remove a macaroon ID using the specified root key ID. For example:
+
+	lncli deletemacaroonid 1
+
+	Note that the default root key ID 0 cannot be deleted.
```

Comment:
Need to add a warning here that all macaroons created from that root key are invalidated by deleting the ID.

Reply:
added